docs/operate/reference/module-configuration.md

```diff
@@ -290,3 +290,69 @@ To set the path to a program or library on a machine, you can set a system varia
 ```
 
 {{% /expand%}}
+
+## Configure an unlisted module
+
+To configure a module that is uploaded to the Viam Registry but has [visibility](/operate/get-started/other-hardware/manage-modules/#change-module-visibility) set to **Unlisted**, you need to manually add the module to your configuration:
+
+1. Navigate to the module's page in the Viam Registry, using the link to the module.
+
+1. Find the **Unlisted module usage** section.
+
+1. Copy the module configuration JSON snippet.
+
+1. In the Viam app, navigate to the **CONFIGURE** tab of the machine you want to configure.
+
+1. Switch to **JSON** mode.
+
+1. Paste the copied module configuration into your `modules` array.
+
+1. Copy the model configuration snippet for the model you want to use, and add it to your `components` or `services` array (as appropriate).
+   For example:
+
+   {{< tabs >}}
+   {{% tab name="Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+"components": [
+  {
+    "name": "sensor-1",
+    "api": "rdk:component:sensor",
+    "model": "jessamy:hello-world:hello-camera",
+    "attributes": {}
+  }
+],
```

Comment:
I think add these to indicate you're not replacing other components/services that may be in this array
```suggestion
  },
  ...
],
```